src/components/TodoApp.tsx

```diff
@@ -0,0 +1,101 @@
+import React, { useContext, useEffect, useMemo, useRef, useState } from 'react';
+import { TodosContext } from './TodosContext';
+import { TodoList } from './TodoList';
+import { TodosFilter } from './TodosFilter';
+import { USER_ID } from '../api/todos';
+import classNames from 'classnames';
+
+export const TodoApp: React.FC = () => {
+  const context = useContext(TodosContext);
+  const { todos, toggleAll, addTodo, handleDeleteCompleted } = context;
```

Comment:
```suggestion
  const { todos, toggleAll, addTodo, handleDeleteCompleted } = useContext(TodosContext);;
```

Reply:
the line will be longer than 80 characters and there will be a linter error